waku/v2/protocol/filter/client.go

```diff
@@ -263,11 +269,11 @@ func (wf *WakuFilterLightNode) FilterSubscription(peerID peer.ID, contentFilter
 		return nil, err
 	}
 
-	if !wf.subscriptions.Has(peerID, contentFilter.Topic, contentFilter.ContentTopics...) {
+	if !wf.subscriptions.Has(peerID, contentFilter.PubsubTopic, maps.Keys(contentFilter.ContentTopics)...) {
```

Comment:
Wondering if we could add a function like this:

```
func (cf ContentFilter) ContentTopics() []string {
   return map.Keys(cf.ContentTopics)
}
```
And call that function directly instead of using `map.Keys` here  and in line 181 and 353?

Reply:
Good idea, thanks, done.